src/authenticated/member/MemberProfile.js

```diff
@@ -113,14 +114,14 @@ const MemberProfile = ({ user, onFormDataChange, editing }) => {
         <div className="Member-skills-left">
           <h2>Projects</h2>
```

Comment:
Let's change the title here to clarify that these are projects that the user is a champion of. We may want to have a separate field in the future for just project involvement. Do you think just `Projects Championed` is adequate?

Reply:
That works for me